db/migrate/20160607141740_remove_ruby_rep_tables_and_triggers.rb

```diff
@@ -0,0 +1,38 @@
+class RemoveRubyRepTablesAndTriggers < ActiveRecord::Migration[5.0]
+  include MigrationHelper
+
+  PREFIX = "rr#{ApplicationRecord.my_region_number}_".freeze
+
+  TRIGGER_QUERY = <<-SQL.freeze
+    SELECT relname, array_agg(tgname) AS triggers
+    FROM
+      pg_trigger t JOIN
+      pg_class c ON t.tgrelid = c.oid
+    WHERE
+      t.tgname like '#{PREFIX}%'
+    GROUP BY relname
```

Comment:
Minor, but can you reorg this as:

```sql
    SELECT relname, array_agg(tgname) AS triggers
    FROM pg_trigger t
      JOIN pg_class c ON t.tgrelid = c.oid
    WHERE t.tgname like '#{PREFIX}%'
    GROUP BY relname
```

The current alignment seems so strange to me.